dist/tools/codespell/check.sh

```diff
@@ -10,7 +10,9 @@ CODESPELL_OPT=" -c"
 CODESPELL_OPT+=" -q 2"
 CODESPELL_OPT+=" --check-hidden"
 CODESPELL_OPT+=" --ignore-words ${TOOLS}/codespell/ignored_words.txt"
-CODESPELL_OPT+=" --skip=RIOT,dist,./wifi-subsys/build,.git,css,html,js,m4a.doxyfile"
+CODESPELL_OPT+=" --skip=RIOT,dist,build,bin,.git,css,html,js,m4a.doxyfile"
```

Comment:
Here is the error, i change this to:
- build -> to avoid the compiled elements by subsystem
- bin -> to avoid the compiled elements by firmware